Lib/sam.h

```diff
@@ -41,9 +48,1141 @@
 
 // ----- Macros -----
 
+#define SOFTWARE_RESET NOP //TODO: Lookup macro
```

Comment:
https://github.com/kiibohd/controller/blob/master/Lib/kinetis.h#L4430

Just need to do a soft reset on the chip.